src/controllers/commands/CommandController.cpp

```diff
@@ -538,6 +539,31 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand(
+        "/streamlink", [](const QStringList &words, ChannelPtr channel) {
```

Comment:
I think `/streamlink` without any parameters should be legal when used in Twitch channels.
Using `/streamlink pajlada` should be legal in any channel, not just Twitch channels.